pkg/cassandra/session.go

```
@@ -190,6 +190,19 @@ func isSelect(query string) bool {
 	return true
 }
 
+// lastN returns the last n entries of s, or everything when s is shorter than
+// that. The order of the entries is preserved.
+func lastN(s []string, n int) []string {
```

golangci-lint (2.12.2)

[error] func lastN is unused

(unused)